honeybee_grasshopper_radiance/src/HB Annual Glare Metrics.py

```diff
@@ -0,0 +1,136 @@
+        grid_filter_: The name of a grid or a pattern to filter the grids. For instance,
+            first_floor_* will simulate only the sensor grids that have an
+            identifier that starts with first_floor_. By default all the grids
+            will be processed.
+        mesh_: An optional list of Meshes that align with the meshes used in the imageless
+            annual glare recipe, which will be used to assign an area to each sensor.
+            If no mesh is connected here, it will be assumed that each sensor represents
+            an equal area to all of the others.
+        _target_time_: A minimum threshold of occupied time (eg. 95% of the time), above
+            which a given sensor passes and contributes to the spatial glare
+            autonomy. (Default: 95%).
+
+    Returns:
+        report: Reports, errors, warnings, etc.
+        GA: Glare autonomy results in percent. GA is the percentage of occupied hours
+            that each sensor is free of glare according to the DGP threshold.
+            Each value is for a different sensor of the grid. These can be plugged
+            into the "LB Spatial Heatmap" component along with meshes of the sensor
+            grids to visualize results.
+        sGA: Spatial glare autonomy as percentage of area for each analysis grid.
+        pass_fail: A data tree of zeros and ones, which indicate whether a given senor
+            passes the criteria for being free of glare (1) or fails the criteria (0). 
+            Each value is for a different sensor of the grid. These can be plugged
+            into the "LB Spatial Heatmap" component along with meshes of the
+            sensor grids to visualize results.
+"""
+
+ghenv.Component.Name = "HB Annual Glare Metrics"
+ghenv.Component.NickName = 'GlareMetrics'
+ghenv.Component.Message = '1.4.0'
+ghenv.Component.Category = 'HB-Radiance'
+ghenv.Component.SubCategory = '4 :: Results'
+ghenv.Component.AdditionalHelpFromDocStrings = '1'
+
+import os
+
+try:
+    from ladybug.datacollection import BaseCollection
+except ImportError as e:
+    raise ImportError('\nFailed to import ladybug:\n\t{}'.format(e))
+
+try:
+    from honeybee_radiance.postprocess.annualglare import glare_autonomy_from_folder
+except ImportError as e:
+    raise ImportError('\nFailed to import honeybee_radiance:\n\t{}'.format(e))
+
+try:
+    from honeybee_energy.lib.schedules import schedule_by_identifier
+except ImportError as e:  # honeybee schedule library is not available
+    schedule_by_identifier = None
+
+try:
+    from ladybug_rhino.grasshopper import all_required_inputs, list_to_data_tree, \
+        data_tree_to_list
+except ImportError as e:
+    raise ImportError('\nFailed to import ladybug_rhino:\n\t{}'.format(e))
+
+
+if all_required_inputs(ghenv.Component):
+    # set default values for the thresholds and the grid filter
+    grid_filter_ = '*' if grid_filter_ is None else grid_filter_
+    _threshold_ = _threshold_ if _threshold_ else 0.4
```

Comment:
For consistency with the recipe, could we call this input `_glare_tresh_` instead of `_threshold_`. Particularly, in this case, it's a little ambiguous whether "threshold" refers to a Glare/DGP threshold or a threshold of acceptable glare time.